intakebuilder/getinfo.py

```diff
@@ -131,8 +131,12 @@ def getInfoFromGFDLDRS(dirpath,projectdir,dictInfo):
     for i in range(nlen-1,0,-1):
       try:
```

Comment:
It looks like the code still depends on builderconfig, hence there are errors thrown such as the following. Test this branch commenting out output_path_template and output_file_template because this is "primarily" coming from the config file moving forward 

-

```
Traceback (most recent call last):
  File "./gen_intake_gfdl.py", line 107, in <module>
    main()
  File "/nbhome/Aparna.Radhakrishnan/conda/envs/intake/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/nbhome/Aparna.Radhakrishnan/conda/envs/intake/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/nbhome/Aparna.Radhakrishnan/conda/envs/intake/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/nbhome/Aparna.Radhakrishnan/conda/envs/intake/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "./gen_intake_gfdl.py", line 86, in main
    list_files = gfdlcrawler.crawlLocal(project_dir, dictFilter, dictFilterIgnore, logger, configyaml)
  File "/nbhome/a1r/CatalogBuilder/intakebuilder/gfdlcrawler.py", line 47, in crawlLocal
    dictInfo = getinfo.getInfoFromGFDLDRS(dirpath, projectdir, dictInfo,configyaml)
  File "/nbhome/a1r/CatalogBuilder/intakebuilder/getinfo.py", line 132, in getInfoFromGFDLDRS
    output_path_template = builderconfig.output_path_template 
AttributeError: module 'intakebuilder.builderconfig' has no attribute 'output_path_template'
```

Reply:
Lines 129 - 132 should prevent that from happening. Looks like configyaml isn't being checked correctly here. 
```
 if configyaml:
        output_path_template = configyaml.output_path_template
    else:
        output_path_template = builderconfig.output_path_template 
```

Reply:
gotcha, might be just this line.. https://github.com/aradhakrishnanGFDL/CatalogBuilder/blob/8a7cf47e193a40324aec282bbefdcbb6b60928e7/scripts/gen_intake_gfdl.py#L60